requirements.txt

```diff
@@ -48,4 +48,4 @@ python-social-auth==0.2.12
 boto==2.38.0
 rollbar==0.9.12
 unittest-xml-reporting==2.1.0
-PyMySQL==0.7.9
+PyMySQL==0.8.0
```

Comment:
This isn't necessary but apparently they did fix the `KeyError` that occurs with MySQL 8.  I can revert this if you think it's best.